artifactory/utils/dependenciesutils.go

```diff
@@ -22,6 +22,7 @@ const (
 	// configured to proxy releases.jfrog.io.
 	// This env var should store a server ID and a remote repository in form of '<ServerID>/<RemoteRepo>'
 	ExtractorsRemoteEnv = "JFROG_CLI_EXTRACTORS_REMOTE"
+	releasesRemoteEnv   = "JFROG_CLI_RELEASES_REPO"
```

Comment:
Let's add a full description for this variable.

Reply:
done